src/components/InventoryTable/EntityTable.js

```diff
@@ -199,6 +200,7 @@ EntityTable.defaultProps = {
     hasCheckbox: true,
     showActions: false,
     rows: [],
+    variant: TableVariant.compact,
```

Comment:
Is it fine this way?

Reply:
If you look at line 181, there is already a `variant` prop defined in the `tableProps`. Changing the default for the `tableProps` below should do the trick.

Reply:
Unfortunately changing the ``tableProps`` defaults does nothing, because the default props are overwritten and not merged when the object is not empty.

Example:
When parent sets ``tableProps`` with ``canSelectAll``, default prop ``variant`` is not applied
![image](https://user-images.githubusercontent.com/20592948/158591386-3335e65c-e94e-4018-bdfc-38ce9c8b2cca.png)

But when parent doesnt set ``tableProps``, default prop ``variant`` is applied
![image](https://user-images.githubusercontent.com/20592948/158591105-87aba2f1-e61e-4676-abfe-1ddb47203c47.png)

I've found a fix on line 150 by merging the tableProps with the defaultProps:
from
```{...tableProps}```
to
```{ ...Object.assign({}, EntityTable.defaultProps.tableProps, tableProps) }```

but it doesn't look that good.

Should I leave it as it is currently or is there a better way?

Reply:
You are right. Well, let's leave it as is, this works.